### PR TITLE
chore(nav-list): create active state class for matching 2018 spec update

### LINKED ISSUE #13084: Active style for mat-nav-list
There's a theme-specific active style for navigation items in a nav drawer:

https://material.io/design/components/navigation-drawer.html#standard-drawer

(Note the way "inbox" is highlighted, because that's the section we're currently in).

However, there's no way to achieve this style in Angular Material. Ideally, there'd be a way to do this with routerLinkActive on a mat-nav-list.

### PR DESCRIPTION
**Pull request by josephperrott**
Fixes #13084

### CODE REVIEW

**Comment by crisbeto** [comment]
Doesn't #13084 also call for there to be a rounded outline around the link?

**Comment by josephperrott** [comment]
Ahh, didn't see that.

File: src/lib/list/list.ts [hunk 1]
```diff
@@ -139,6 +172,17 @@ export class MatListItem extends _MatListItemMixinBase implements AfterContentIn
     // TODO: consider turning the setter into a function, it doesn't do anything as a class.
     // tslint:disable-next-line:no-unused-expression
     new MatLineSetter(this._lines, this._element);
+
+    if (this.active) {
+      setTimeout(() => {
```
Comment (jelbourn) [inline src/lib/list/list.ts]: Add a comment that explains why the timeout is used

**Comment by josephperrott** [comment]
@jelbourn Ready for another look

**Comment by Nodarii** [comment]
Hello guys, 

Should we expect this to be merged soon?

**Pull request closed without merging**